### PR TITLE
enrichment toggle and updates

### PR DESCRIPTION
**Pull request by ManuelHanisch**
- Enrichment toggle in backend added.
- Made header links inside README.md github friendly
- A few minor code optimizations.

### CODE REVIEW

File: Omikron/Factfinder/Block/FF/Communication.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function __construct(Template\Context $context, $data = [], Data $helper,
 
         $this->_configData = [
             'url' => [
-                'value' => '/' . Data::FRONT_NAME . '/',
+                'value' =>  ($this->_helper->isEnrichmentEnabled() ? '/' . Data::FRONT_NAME . '/' : $this->_helper->getAddress()),
```
Comment (HJK181) [inline Omikron/Factfinder/Block/FF/Communication.php]: Can you please explain the usage of the Constant `Data::FRONT_NAME`? 
Is it also possible that you install the changes of this PR to our Magento Instance?

File: Omikron/Factfinder/Helper/Data.php [hunk 2]
```diff
@@ -45,10 +46,21 @@ public function __construct(
 
     /**
      * Public Getter
+     * @param null|int|string $scopeCode
```
Comment (HJK181) [inline Omikron/Factfinder/Helper/Data.php]: Why did you rename it to $scopeCode when you pass the store id as parameter? The previous doc name did fit better to the current argument from line 85 Communication.php.

File: Omikron/Factfinder/Helper/Data.php [hunk 3]
```diff
@@ -130,12 +142,12 @@ public function getShowAddToCartButton()
 
     /**
      * Returns the current FACT-Finder version
-     * @param null|string $storeCode
+     * @param null|int|string $scopeCode
```
Comment (HJK181) [inline Omikron/Factfinder/Helper/Data.php]: Why did you rename it to $scopeCode when you pass the store id as parameter? The previous doc name did fit better to the current argument from line 85 Communication.php. 

Also $scopeCode on the other methods.

File: Omikron/Factfinder/Model/Export/Product.php [hunk 4]
```diff
@@ -194,7 +194,7 @@ public function exportProducts($updateFieldRoles = false)
 
         foreach ($stores as $store) {
             $currChannel = $this->helperData->getChannel($store->getId());
-            if (in_array($currChannel, $exportedChannels)) {
+            if (in_array($currChannel, $exportedChannels) || !$this->helperData->isEnabled($store->getId())) {
```
Comment (HJK181) [inline Omikron/Factfinder/Model/Export/Product.php]: It would be helpful if the module is capable to export the data in disabled state because there might be some agencies or customers which don't have a testing environment.

**Comment by HJK181** [comment]
Shouldn't you check here
https://github.com/ManuelHanisch/magento2-module/blob/45540e4fe3c3b78f7a821328541e506fa0bae611/Omikron/Factfinder/Controller/Proxy/Call.php#L72

against the enrichment enabled property or where does the setting have an impact on the result?

File: Omikron/Factfinder/Helper/Data.php (hunk 3 above)
Comment (ManuelHanisch) [inline Omikron/Factfinder/Helper/Data.php]: I switched to naming it scopeCode because scopeConfig->getValue(...) uses it as scopeCode too and i tried to align the naming to that function.
scopeConfig->getValue(...) gets the value from the database table "core_config_data".  This table uses the storeId for its field "scope_id". 

public function getValue($path, $scopeType = ScopeConfigInterface::SCOPE_TYPE_DEFAULT, $scopeCode = null);

On the other hand storeCode or storeId (or even scopeId) might be better understandable...I decided in the end for scopeCode, but there is nothing wrong with the other names.
